#files
install.packages("nortest")
library(sm)
library(vioplot)
library(ggplot2)
library(nortest)
setwd("/home/ross/Dropbox/a_thesis/github/thesis_ESM")
x <- read.csv("long_tail_phylogen_articles_data.csv")
colnames(x) <- c("journal","number")
top50 <- subset(x, number > 313)
restof <- subset(x, number < 313)
attach(x)
names(x)
nam1 <- as.character(seq(51,500,by=1))
top50$journal <- factor(top50$journal, levels=c("INTERNATIONAL JOURNAL OF SYSTEMATIC AND EVOLUTIONARY MICROBIOLOGY","MOLECULAR PHYLOGENETICS AND EVOLUTION","MOLECULAR BIOLOGY AND EVOLUTION","APPLIED AND ENVIRONMENTAL MICROBIOLOGY","PLOS ONE","ZOOTAXA","PROCEEDINGS OF THE NATIONAL ACADEMY OF SCIENCES OF THE UNITED STATES OF AMERICA","BMC EVOLUTIONARY BIOLOGY","MOLECULAR ECOLOGY","JOURNAL OF CLINICAL MICROBIOLOGY","JOURNAL OF MOLECULAR EVOLUTION","GENE","AMERICAN JOURNAL OF BOTANY","ARCHIVES OF VIROLOGY","BIOLOGICAL JOURNAL OF THE LINNEAN SOCIETY","JOURNAL OF GENERAL VIROLOGY","JOURNAL OF VIROLOGY","SYSTEMATIC BIOLOGY","EVOLUTION","MYCOLOGIA","SYSTEMATIC BOTANY","PLANT SYSTEMATICS AND EVOLUTION","ENVIRONMENTAL MICROBIOLOGY","JOURNAL OF MEDICAL VIROLOGY","TAXON","NUCLEIC ACIDS RESEARCH","BMC GENOMICS","MYCOLOGICAL RESEARCH","BIOINFORMATICS","VIRUS GENES","VIRUS RESEARCH","ZOOLOGICAL JOURNAL OF THE LINNEAN SOCIETY","FEMS MICROBIOLOGY ECOLOGY","JOURNAL OF VERTEBRATE PALEONTOLOGY","JOURNAL OF PHYCOLOGY","JOURNAL OF BACTERIOLOGY","PROCEEDINGS OF THE ROYAL SOCIETY B BIOLOGICAL SCIENCES","SYSTEMATIC AND APPLIED MICROBIOLOGY","FEMS MICROBIOLOGY LETTERS","JOURNAL OF BIOGEOGRAPHY","INTERNATIONAL JOURNAL OF PLANT SCIENCES","VETERINARY MICROBIOLOGY","VIROLOGY","AIDS RESEARCH AND HUMAN RETROVIRUSES","JOURNAL OF BIOLOGICAL CHEMISTRY","ZOOLOGICA SCRIPTA","BOTANICAL JOURNAL OF THE LINNEAN SOCIETY","BMC BIOINFORMATICS","JOURNAL OF EVOLUTIONARY BIOLOGY","NATURE"))
plot(x)
plot(top50,pch=3,main="Distribution of phylogen* articles 2000-2011",xlim=c(1,50),xlab="Top 50 journals containing phylogen*",ylab="Number of phylogen* articles",xaxt="n",yaxt="n")
axis(1,cex.axis=0.8,at=c(1,5,10,15,20,25,30,35,40,45,50))
axis(2,cex.axis=0.8,las=1)
text(x = 23, y = 3580, label = "                      INTERNATIONAL JOURNAL OF SYSTEMATIC AND EVOLUTIONARY MICROBIOLOGY",cex=0.5)
text(x = 15, y = 2790, label = "                      MOLECULAR PHYLOGENETICS AND EVOLUTION", cex=0.5)
text(x = 1, y = 1300, srt=45, label = "MBE", cex=0.5)
text(x = 17, y = 1400, label = "APPLIED ENVIRONMENTAL MICROBIOLOGY", cex=0.5)
text(x = 2, y = 1050, srt=45, label = "PLOS ONE", cex=0.5)
text(x = 3, y = 950, srt=45, label = "ZOOTAXA", cex=0.5)

plot(restof$number,xaxt="n",yaxt="n",main="The rest of the journal distribution",ylab="Number of phylogen* articles", xlab="Journals 51 to 500")
axis(1,cex.axis=0.8,at=c(1,50,100,150,200,250,300,350,400,450),labels=c("51","100","150","200","250","300","350","400","450","500"))
axis(2,cex.axis=0.8,las=1)

hist(x$number)
